data/fixtures/scopes/zig/type.variable.scope

```diff
@@ -0,0 +1,24 @@
+const foo: u8 = 0;
+---
+
+[Content] = 0:11-0:13
+             >--<
+0| const foo: u8 = 0;
+
+[Removal] = 0:11-0:14
+             >---<
+0| const foo: u8 = 0;
```

Comment:
Removal range should be `: u8`

https://github.com/cursorless-dev/cursorless/blob/d21522be750a55b70b50ac251300bfd6dd4baace/data/fixtures/scopes/typescript.core/type/type.variable2.scope#L10